blocks/bricks/wrappers.py

```diff
@@ -1,3 +1,4 @@
+from six.moves import range
```

Comment:
I think we use `xrange` in most other places (they are exactly the same, functionally, but the 'x' is a nice reminder that it's a fake-range i.e. Python 2 `xrange` a.k.a. Python 3 `range`).